frontend/src/types/item.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Item } from "../pages/weights"
+
+export type ItemsResponse = {
```

Comment:
This is called `PaginatedResult` because it can be items or any other stuff in a pagination. Change Item type to use Generic. You can check out `PaginatedResult` in query backend.